docs/faq.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Troubleshooting & FAQ
+---
+
+### Where do I find the DBOS tables?
+
+DBOS checkpoints information about your workflows in an isolated _system database_ in your Postgres database server.
+By default, the name of your system database is your application database name suffixed with `_dbos_sys`.
+For example, if your application database is `dbos_app_starter`, your system database is `dbos_app_starter_dbos_sys`.
+You can connect to and explore your system database with popular database clients like [psql](https://www.postgresql.org/docs/current/app-psql.html) and [DBeaver](https://dbeaver.io/).
+All system database tables are documented [here](./explanations/system-tables.md).
```

Comment:
Also mention that DBOS tables are under the `dbos` schema. So run `\dt dbos.*` to list all the tables. That was asked a lot too.